### PR TITLE
Docs/refactor

### PR DESCRIPTION
**Pull request by joshLong145**
-- Fixes to sidebar modules list and layout
-- Update to schema docs (wip)

## Changes
* Item 1
* Item 2

## Checklist
* [ ] Unit tests
* [ ] Documentation

## References _(optional)_

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202528620406598/1202532730445368) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by mcjcloud — APPROVED** [review]
lgtm

File: docs/src/css/custom.css [hunk 1]
```diff
@@ -3,77 +3,7 @@
...
-    font-style: normal;
-    font-display: swap;
-}
-
-@font-face {
-    font-family: 'THICCCBOI';
-    src: url('/fonts/THICCCBOI-Thin.woff2') format('woff2');
-    font-weight: 100;
-    font-style: normal;
-    font-display: swap;
```
Comment (ntindle) [inline docs/src/css/custom.css:27]: Can we pretty please fix the fonts

File: docs/articles/highway-sdk/objects.md [hunk 2]
```diff
@@ -1,57 +0,0 @@
----
```
Comment (ntindle) [inline docs/articles/highway-sdk/objects.md:1]: Do we know why these files were removed?

File: docs/src/css/custom.css (hunk 1 above)
Comment (joshLong145) [reply docs/src/css/custom.css:27]: wow i didnt check the history on this file, added the font faces back and we now are rendering correctly

File: docs/articles/highway-sdk/objects.md (hunk 2 above)
Comment (joshLong145) [reply docs/articles/highway-sdk/objects.md:1]: they where not really applicable at this time and the information is now within each module

**Review by prdlk — APPROVED** [review]

**Pull request merged**